sw/device/silicon_creator/mask_rom/meson.build

```diff
@@ -137,12 +137,12 @@ foreach device_name, device_lib : sw_lib_arch_core_devices
   )
 endforeach
 
-test('sw_silicon_creator_mask_rom_rsa_verify_unittest', executable(
-    'sw_silicon_creator_mask_rom_rsa_verify_unittest',
+test('sw_silicon_creator_mask_rom_sigverify_mod_exp_ibex_unittest', executable(
```

Comment:
Should the test cases and mock library also move with the sigverify module?

Reply:
Yes, please see #6982 (last 3 commits), this PR is mostly renames and other cleanups to keep its size more manageable.